larpandoracontent/LArVertex/TrainedVertexSelectionAlgorithm.cc

```diff
@@ -296,8 +296,8 @@ void TrainedVertexSelectionAlgorithm::GetEventShapeFeatures(const ClusterList &c
     }
 
     const float xSpan(this->GetCoordinateSpan(xMax, xMin));
-    const float ySpan(this->GetCoordinateSpan(yMax, zMin));
-    const float zSpan(this->GetCoordinateSpan(yMax, zMin));
+    const float ySpan(this->GetCoordinateSpan(yMax, yMin));
+    const float zSpan(this->GetCoordinateSpan(zMax, zMin));
```

Comment:
This looks sound, thank you, but opens up some further discussion about other (existing) implementation, below.